packages/persisted-query-lists/README.md

```diff
@@ -1,3 +1,118 @@
 # `@apollo/persisted-query-lists`
 
-Part of an upcoming Apollo GraphOS feature.
+## Setup
+
+Install the package:
+
+```sh
+npm install @apollo/persisted-query-lists
+```
+
+## Usage
+
+### `generatePersistedQueryIdsAtRuntime`
+
+Helper function passed to [Persisted Query Link](https://www.apollographql.com/docs/react/api/link/persisted-queries)
+that generates query hashes at runtime without the use of a manifest file.
+
+```ts
+import { generatePersistedQueryIdsAtRuntime } from "@apollo/persisted-query-lists";
+import { createPersistedQueryLink } from "@apollo/client/link/persisted-queries";
+import { sha256 } from "crypto-hash";
+
+const persistedQueryLink = createPersistedQueryLink(
+  generatePersistedQueryIdsAtRuntime({ sha256 }),
+);
+```
+
+#### Options
+
+- `sha256`: A SHA-256 hashing function. Can be sync or async. Providing a SHA-256
+  hashing function is required.
+
+### `generatePersistedQueryIdsFromManifest`
+
+Helper function passed to [Persisted Query Link](https://www.apollographql.com/docs/react/api/link/persisted-queries)
+that will read from your manifest configuration to get the persisted query ID.
+See the [`@apollo/generate-persisted-query-manifest`](https://www.npmjs.com/package/@apollo/generate-persisted-query-manifest)
+package to learn how to generate the manifest file.
+
+```ts
+import { generatePersistedQueryIdsFromManifest } from "@apollo/persisted-query-lists";
+import { createPersistedQueryLink } from "@apollo/client/link/persisted-queries";
+
+const persistedQueryLink = createPersistedQueryLink(
+  generatePersistedQueryIdsFromManifest({
+    loadManifest: () => import("./path/to/persisted-query-manifest.json"),
+  }),
+);
+```
+
+#### Options
+
+- `loadManifest`: A function that returns your persisted query manifest
+  configuration. Can by sync or async. We recommend using a dynamic import to
+  avoid bundling the manifest configuration with your production build.
+
+### `createPersistedQueryManifestVerificationLink`
+
+An [Apollo Link](https://www.apollographql.com/docs/react/api/link/introduction)
+that verifies that queries sent to your server can be matched to your manifest
+configuration. See the [`@apollo/generate-persisted-query-manifest`](https://www.npmjs.com/package/@apollo/generate-persisted-query-manifest)
+package to learn how to generate the manifest file.
+
+NOTE: This link is not a terminating link and will forward the operation through
+the link chain. We recommend using the [Persisted Query Link](https://www.apollographql.com/docs/react/api/link/persisted-queries)
```

Comment:
The PQ link is not a terminating link.

And I don't think this is too likely to be used with the PQ link anyway — it's more of a "before you're ready to start sending PQs, use this to verify that sending PQs would be a good idea" thing.

Reply:
Oh duh you're right. Apparently I fell asleep while I was writing these docs 😅

Reply:
I ended up just removing the second sentence. Let me know if you think the first paragraph is enough to capture the "before you're ready to start sending PQs" sentiment.